.github/workflows/push-deploy-website.yml

```
@@ -0,0 +1,57 @@
+name: PUSH Workflow - Website Deployment
+
+on:
+  push:
+    branches:
+      - 'develop'
+    paths:
+      - docs/**
+
+env:
+  CODECOV_UNIQUE_NAME: CODECOV_UNIQUE_NAME-${{ github.run_id }}-${{ github.run_number }}
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)